suite-native/state/src/extraDependencies.ts

```diff
@@ -42,24 +42,32 @@ const createEmptyReducer = (name: string) => (_state: any, action: any) => {
 
 export const extraDependencies: ExtraDependencies = {
     thunks: {
-        fetchAndUpdateAccount: createEmptyThunk('fetchAndUpdateAccount'),
         notificationsAddEvent: createEmptyThunk('notificationsAddEvent'),
     },
     selectors: {
         selectFeeInfo: (networkSymbol: any) =>
             createEmptySelector('selectFeeInfo', testMocks.fee, { networkSymbol }),
-        selectAccounts: createEmptySelector('selectAccounts', []),
         selectDevices: createEmptySelector('selectDevices', []),
         selectBitcoinAmountUnit: createEmptySelector(
             'selectBitcoinAmountUnit',
             PROTO.AmountUnit.BITCOIN,
         ),
+        selectEnabledNetworks: createEmptySelector('selectEnabledNetworks', []),
+        selectAccountTransactions: createEmptySelector('selectAccountTransactions', {
```

Comment:
this is little bit confusing `mockedTransaction` key in selectTransaction should be actually `accountHash` so I would prefer something like `accountHashMockedKey` or something like that.